cardano-testnet/src/Testnet/Blockfrost.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
I also have to request the remvoal of `RecordWildCards` as it reduces the readability of the code. There is one exception we allow, that is in `cardano-api` when dealing with the protocol parameters types that have lots of fields. However I see you using it with a record type that only has 2 fields.